src/Core/MailTemplates/Handlebars/SecretsManagerAccessRequest.text.hbs

```diff
@@ -0,0 +1,21 @@
+{{#>FullTextLayout}}
+
+{{{UserNameRequestingAccess}}} has requested access to secrets manager:
+
+============
+
+{{{EmailContent}}} - {{{UserNameRequestingAccess}}}
+
+============
+
+Contact Bitwarden
+
+============
+
+{{WebVaultUrl}}/?utm_source=sm_request_access_email&utm_medium=email
+
+============
+
```

Comment:
`WebVaultUrl` isn't a parameter in the template. 

I think this would just be 

```suggestion

============

Contact Bitwarden (https://bitwarden.com/contact-sales/?utm_source=sm_request_access_email&utm_medium=email)

============

```